### ISSUE TITLE
Option \tikzset{annotate equations/arrow/.style={->}} does not work

### ISSUE THREAD

**Issue by qscas** [st--/annotate-equations#19]
environment : `overleaf`  
compiler=`lualatex` or `xelatex`

```
\begin{equation*}
    \eqnmarkbox[blue]{node1}{e_q^n}
    \eqnmark[red]{node2}{f(x)}
    \tikzmarknode{node3}{kT}
\end{equation*}
\tikzset{annotate equations/arrow/.style={->}}
\annotatetwo[yshift=1em]{above}{node1}{node3}{text}
%\annotate[yshift=1em]{left}{node1}{my}
\annotate[yshift=-0.5em]{below, left}{node2}{annotation}
\annotate[yshift=-0.1em]{below, label below}{node3}{text}
```
though I changed the default settings, the arrow lines compiled still keep the original style. `\tikzset{annotate equations/arrow/. style = {color=ForestGreen, >=latex', very thick, dashed}}` also has no effect.

![Snipaste_2023-07-13_10-23-21](https://github.com/st--/annotate-equations/assets/107034893/979a7306-9ad0-4e8f-aa2a-977bced7ef7c)

**Comment by st--** [comment]
What version of the package are you using? I've just tried to reproduce your issue and the `\tikzset` works fine for me.

Side notes:

1. The way I coded up the arrows, they go from equation to annotation, so you probably want `<-` instead of `->` in your first example.
2. In general it's helpful to provide a complete, self-contained and compilable [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example). For example, I had to add the `\documentclass{article} \begin{document} ... \end{document}` boiler plate as well as including the `annotate-equations` package as well as `amsmath` for `equations*` and change the color to just `green` because I couldn't be bothered to also figure out the correct xcolor options... So it wasn't too bad this time, but generally the easier you make it for the author of a package to reproduce their issue, the more capacity they have left to actually consider how to fix it:)
